lib/xclarity_client/services/update_repo_management.rb

```diff
@@ -2,12 +2,12 @@ module XClarityClient
   class UpdateRepoManagement < Services::XClarityService
     manages_endpoint UpdateRepo
 
-    def population(opts = {})
+    def fetch_all(opts = {})
       raise "Option key must be provided for update_repo resource" if opts.empty?
       raise "Option key must be provided for update_repo resource" if not (opts.has_key?(:key) || opts.has_key?("key"))
       repoKey = opts[:key] || opts["key"]
       if repoKey == "supportedMts" || repoKey == "size" || repoKey == "lastRefreshed" || repoKey == "importDir" || repoKey == "publicKeys" || repoKey == "updates" || repoKey == "updatesByMt" || repoKey == "updatesByMtByComp"
```

Comment:
Could you use something like `["supportMts", "size", (...), "updatesByMtByComp"].include? repoKey` instead of using lots of "or" operators? It will be more legible and make the line shorter.

Reply:
Yes, it is a good idea, but I think that could be done in another PR to don't loose the focus of this one. We have a great amout of codeclimate issues, the idea is to create others PRs to solve it.

Reply:
It will not be a focus change, once the code climate notified it as an issue.